.github/PULL_REQUEST_TEMPLATE

```diff
@@ -3,7 +3,7 @@
 - [ ] Add a test case for the new plugin
 - [ ] Add a CHANGELOG entry for the new plugin
 - [ ] Add a component id in [the main repo](https://github.com/apache/skywalking/blob/master/oap-server/server-bootstrap/src/main/resources/component-libraries.yml#L415)
```

Comment:
This link seems 404 now. 
Maybe it should be changed to ``https://github.com/apache/skywalking/blob/master/oap-server/server-starter/src/main/resources/component-libraries.yml``